packages/ui/package.json

```diff
@@ -8,6 +8,7 @@
       "import": "./dist/esm/index.js",
       "require": "./dist/index.js"
     },
+    "./package.json": "./package.json",
```

Comment:
Why was this added?

Reply:
It's a requirement for Metro bundler (React Native bundler)